src/3D/LandBlock.h

```diff
@@ -72,8 +74,6 @@ class LandBlock
 	std::unique_ptr<btBvhTriangleMeshShape> _physicsMesh;
 	std::unique_ptr<btRigidBody> _rigidBody;
 
-	const bgfx::Memory* BuildVertexList(LandIsland& island);
-
-	friend LandIsland;
```

Comment:
Nice that we can get rid of this friend

Reply:
Friend is a weird stuff in cpp....
Wasn't expecting such a weird thing!